docs/articles/sdk/installing-the-sdk.rst

```diff
@@ -10,18 +10,19 @@
 Installing the SDK
 ------------------
 
-The SDK file is not executable by default. This needs to be changed with ``chmod``. If you downloaded
-the SDK from the internet, the ``<SDK location>`` is typically ``~/Downloads``. If you built the SDK from build directory, it can be found on ``tmp/deploy/sdk`` 
+The SDK file is not executable by default. This needs to be changed with ``chmod``. If you
+downloaded the SDK from the internet, the ``<SDK location>`` is typically ``~/Downloads``. If
+you built the SDK from build directory, it can be found on ``tmp/deploy/sdk``.
```

Comment:
(Maybe disregard this comment since you did not actually change anything here except line breaks... ?)

- Nit: capital I on Internet?
- "built the SDK from build directory" is missing a "the"?
- s/found on/found in/